### PR TITLE
Make TGUI things look gooder

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
gooder look

## Changelog
:cl:
add: Better looking TGUI uis
fix: Stopped href vending of hidden products in vending machines without hacking them first
tweak: Most vendors will now vend things into your hands if possible
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
image
image
image
image

**Comment by Rohesie** [comment]
Blessed.

**Comment by Timmeye** [comment]
that looks a lot gooder

**Comment by spookydonut** [comment]
image
image
image

**Comment by spookydonut** [comment]
image

File: code/game/objects/machinery/vending/vending.dm [hunk 1]
```diff
@@ -508,7 +512,9 @@ GLOBAL_LIST_INIT(vending_white_items, typecacheof(list(
 			src.speak(src.vend_reply)
 			src.last_reply = world.time
 
-	release_item(R, vend_delay)
+	var/obj/item/new_item = release_item(R, vend_delay)
+	if(istype(new_item))
+		user.put_in_any_hand_if_possible(new_item, warning = FALSE)
```
Comment (psykzz) [inline code/game/objects/machinery/vending/vending.dm:517]: Internally this called equip to slot, which seems to apply armor from the item to the mob.
This caused some issues where marines were stacking double armor and becoming invulnerable.

Im not sure exactly why this is happening here and not elsewhere, but if vend essential, heavy helmet etc, and check your chest limb you'll see around 65 armor before you equip any gear.

Comment (spookydonut) [reply code/game/objects/machinery/vending/vending.dm:517]: Sounds like a problem with armor code not with this

**Pull request merged**